acapy_wallet_groups_plugin/__init__.py

```diff
@@ -3,7 +3,7 @@
 from aries_cloudagent.admin.request_context import InjectionContext
 from aries_cloudagent.wallet.models.wallet_record import WalletRecord
 
-__version__ = "0.1.0"
+__version__ = "0.1.7"
```

Comment:
Seems like a high version bump :).

Reply:
guess it's good to keep it consistent with the version in toml